src/opm/parser/eclipse/EclipseState/Schedule/WellTestState.cpp

```diff
@@ -133,6 +133,20 @@ namespace Opm {
         return output;
     }
 
+    double WellTestState::lastTestTime(const std::string& well_name) const {
+        const auto well_iter = std::find_if(wells.begin(),
+                                            wells.end(),
+                                            [&well_name](const ClosedWell& well)
+                                            {
+                                                return (well.name == well_name);
+                                            });
+        if (well_iter == wells.end()) {
+            return InvalidTime;
+        } else {
+            return well_iter->last_test;
+        }
+    }
```

Comment:
There is a `hasWell()` method - I would very much prefer that the `lastTestTime()` raised an exception instead of returning a bogus value - at least the downstream code will be nicer that way.

Reply:
> There is a `hasWell()` method

Any chance of upgrading that to `skyLake()`?

:wink:

Reply:
The downstream code already called hasWell(), so it is guaranteed to not fail. I'll change to throwing.